### PR TITLE
Add disclaimers to zones

### PR DESCRIPTION
**Pull request by pierresegonne**
## Issue

Some zones are missing disclaimers

## Description

Used the following script to automate the US zones (+ `pnpx prettier --write .`)

```python
from ruamel.yaml import YAML

from electricitymap.contrib.config import ZONES_CONFIG

yaml = YAML(typ="safe")
yaml.default_flow_style = False

EXCLUDED_ZONES = [
    "US-CAL-CISO",
    "US-MIDA-PJM",
    "US-CENT-SWPP",
    "US-NY-NYIS",
    "US-NE-ISNE",
    "US-HI-NI",
    "US-HI-KA",
    "US-HI-MO",
    "US-HI-MA",
    "US-HI-HA",
    "US-HI-KH",
    "US-HI-LA",
    "US-FLA-HST",
    "US-AK",
]

US_ZONES = [
    z_k
    for z_k in ZONES_CONFIG.keys()
    if z_k.startswith("US-") and z_k not in EXCLUDED_ZONES
]

# Read config/zones/${zone_key}.json and add disclaimer "Data for real-time consumption and generation mix is estimated. Real-time exchanges are not available."
for zone_key in US_ZONES:
    zone_config = yaml.load(
        open(f"config/zones/{zone_key}.yaml", "r", encoding="utf-8"),
    )
    zone_config[
        "disclaimer"
    ] = "Data for real-time consumption and generation mix is estimated. Real-time exchanges are not available."
    with open(f"config/zones/{zone_key}.yaml", "w", encoding="utf-8") as f:
        yaml.dump(zone_config, f)

```

### Double check

- [x] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

File: config/zones/US-SW-EPE.yaml [hunk 1]
```diff
@@ -172,74 +174,74 @@ fallbackZoneMixes:
       datetime: '2017-01-01'
       value:
         battery discharge: 0
-        biomass: 0.000003853100112863885
+        biomass: 3.853100112863885e-06
```
Comment (pierresegonne) [inline config/zones/US-SW-EPE.yaml:177]: No idea why the formatting changed here

Comment (VIKTORVAV99) [reply config/zones/US-SW-EPE.yaml:177]: Prettier or the Yaml package, It should be fine either way.

Comment (madsnedergaard) [inline config/zones/US-SW-EPE.yaml:177]: Why were all these numbers updated? Please do it in a separate PR so we can keep track of things and point to a specific PR in future :)

**Review by madsnedergaard — CHANGES_REQUESTED** [review]
Nice! A few comments, most important is the comment about changing of emission factor values

File: config/zones/US-SW-EPE.yaml (hunk 1 above)
Comment (VIKTORVAV99) [reply config/zones/US-SW-EPE.yaml:177]: The value is the same but I think the Yamel package or prettier formatted them.

**Comment by pierresegonne** [comment]
Formatting issues have been fixed

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

**Pull request merged**